WinUIGallery/Package.appxmanifest

```diff
@@ -10,20 +10,20 @@
   <Identity Name="Microsoft.WinUI3ControlsGallery" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="2.1.0.0" />
   <mp:PhoneIdentity PhoneProductId="863667e0-667a-4bb4-ac52-c59656c7333a" PhonePublisherId="00000000-0000-0000-0000-000000000000" />
   <Properties>
-    <DisplayName>WinUI 3 Gallery</DisplayName>
+    <DisplayName>WinUI 3 Gallery Dev</DisplayName>
```

Comment:
revert this line.  Debug builds should use the Package.Dev manifest and the dev identity.  Release should use the Package manifest and the non-dev identity so we can publish that build to the store.

Reply:
there's also a dependency on DisplayName in WinUIGalleryDesktopTests.cs